tests/integration/test_studio_editable.py

```diff
@@ -274,13 +275,17 @@ class FancyXBlock(StudioEditableXBlockMixin, XBlock):
     string_multiline = String(display_name="Multiline", multiline_editor=True, allow_reset=False)
     string_multiline_reset = String(display_name="Multiline", multiline_editor=True)
     string_html = String(display_name="Multiline", multiline_editor='html', allow_reset=False)
+    string_with_help = String(
+        display_name="String Field with Help Text",
+        help="Learn more about <a class='field_help_link' href='https://en.wikipedia.org/wiki/Color'>colors</a>."
+    )
     # Note: The HTML editor won't work in Workbench because it depends on Studio's TinyMCE
 
     editable_fields = (
         'bool_normal', 'dict_normal', 'float_normal', 'float_values', 'int_normal', 'int_ranged', 'int_dynamic',
         'int_values', 'list_normal', 'list_intdefs', 'list_strdefs', 'list_set_ints', 'list_set_strings',
         'string_normal', 'string_values', 'string_values_provider', 'string_named', 'string_dynamic',
-        'string_multiline', 'string_multiline_reset', 'string_html',
+        'string_multiline', 'string_multiline_reset', 'string_html', 'string_with_help'
```

Comment:
The `README` talks about `string_multiline` and `string_html` but doesn't have info about `string_multiline_reset` or the new `string_with_help`. Is a `README` update appropriate?